spec/forms/claim_details_form_spec.rb

```diff
@@ -46,13 +46,13 @@
 
   describe '#other_known_claimants' do
     context 'when #other_known_claimant_names is blank' do
```

Comment:
this needs the context name changing - as it is no longer based on other_known_claimant_names